dev-lang/lazarus/lazarus-2.2.4.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop
+
+# See https://wiki.freepascal.org/Lazarus_2.0_fixes_branch for notes and
+# changes in FPCVER. It *does* change between minor versions of lazarus.
+FPCVER="3.2.2"
+
+DESCRIPTION="feature rich visual programming environment emulating Delphi"
+HOMEPAGE="https://www.lazarus-ide.org/"
+SRC_URI="mirror://sourceforge/lazarus/${P}-0.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1-with-linking-exception"
+SLOT="0/2.2" # Note: Slotting Lazarus needs slotting fpc, see DEPEND.
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk2 +gui extras"
+REQUIRED_USE="gtk2? ( gui )
+	extras? ( gui )"
```

Comment:
IIRC REQUIRED_USE is normally one-line `REQUIRED_USE="gtk2? ( gui ) extras? ( gui )"`